samples/sample_common/include/brc_routines.h

```diff
@@ -52,12 +55,14 @@ NalHrdConformance | VuiNalHrdParameters   |  Result
     on (or default)      on (or default)    => MFX_BRC_HRD_STRONG
 --------------------------------------------------------------
 */
+#if !defined(__MFXENCTOOLS_INT_H__)
```

Comment:
This is a wrong change! `_studio/shared/include/mfxenctools-int.h` is mediasdk internal file, not an API header! You just can't refer to it in samples.

Effectively, I don't see that any of the changes you've mode to samples bring any value. They should completely be dropped.